packages/common/src/components/UvText/UvText.tsx

```diff
@@ -205,6 +207,32 @@ const UvText: React.FC<Props> = ({ question }) => {
     return 'Ej angivet'
   }
 
+  const getMedicalInvestigationValue = (valueList: ValueMedicalInvestigationList) => {
+    return (
+      <table className="ic-table iu-fullwidth">
+        <thead>
+          <tr>
+            <th scope="col">Ange utredning eller underlag</th>
+            <th scope="col">Datum</th>
+            <th scope="col">Från vilken vårdgivare</th>
```

Comment:
This is different in angular. Should it be changed?

Angular version: "Från vilken vårdgivare kan Försäkringskassan hämta information om utredningen/underlaget?"

Reply:
We just discussed this with Inera and agreed to shrink this text to "Vårdgivare", i'll correct that